setup.py

```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env python
 import sys
-from setuptools import setup
+from setuptools import setup, Distribution
 from setuptools.command.test import test as TestCommand
 from sys import platform
 from platform import architecture
-import shutil
 
-if platform == 'win32' and architecture()[0] == '32bit':
-    shutil.copy2('win/sndfile32.dll', 'win/sndfile.dll')
-    sndfile = [('', ['win/sndfile.dll', 'win/sndfile_license'])]
-elif platform == 'win32' and architecture()[0] == '64bit':
-    shutil.copy2('win/sndfile64.dll', 'win/sndfile.dll')
-    sndfile = [('', ['win/sndfile.dll', 'win/sndfile_license'])]
+if platform in ('darwin', 'win32'):
+    packages = ['pysoundfile_binaries']
+    package_data = {'pysoundfile_binaries':
+                    ['pysoundfile_binaries/libsndfile_license']}
+    if platform == 'darwin':
+        package_data['pysoundfile_binaries'] += \
+            ['pysoundfile_binaries/libsndfile_darwin.dylib']
+    elif platform == 'win32':
+        package_data['pysoundfile_binaries'] += \
+            ['pysoundfile_binaries/libsndfile_win_{}.dll'
```

Comment:
This would work for `libsndfile_win_64bit.dll`, but the file is named `libsndfile_win64.dll`!